arm/src/transaction.rs

```diff
@@ -117,4 +186,60 @@ mod tests {
     fn test_transaction() {
         let _ = generate_test_transaction(1);
     }
+
+    #[test]
+    #[cfg(feature = "aggregation")]
+    fn test_aggregation_works() {
+        let tx = generate_test_transaction(1);
+
+        for strategy in vec![AggregationStrategy::Sequential, AggregationStrategy::Batch] {
+            let mut tx_str = tx.clone();
+            tx_str.aggregate_with_strategy(strategy.clone());
+            assert_eq!(true, tx_str.aggregation_proof.is_some());
+            assert_eq!(true, tx_str.verify_aggregation().is_some());
```

Comment:
seems it can be simplified to: 
tx_str.aggregation_proof.unwrap(), or
assert!(tx_str.aggregation_proof.is_some())